calculator/src/App.js

```diff
@@ -20,13 +20,24 @@ function App() {
 
   const handleSubmit = e => {
     e.preventDefault();
-    calculateSavings();
-    setShowChart(true);
+    if (!initialSavings || !interestRate) {
+      alert("Please enter both your initial savings and bank interest rate.")
+    }  else {
+      calculateSavings();
+    }
   };
 
   const calculateSavings = () => {
     const initialAmount = parseFloat(initialSavings);
     const interestRatePercent = parseFloat(interestRate / 100);
+    console.log(initialAmount)
+    console.log(interestRatePercent)
+
+    if (isNaN(initialAmount) || isNaN(interestRatePercent)) {
+      alert("Please only enter numbers")
+      return
+    }
+
```

Comment:
super minor, but would consider doing a conditional that would handle negative numbers here!